op-e2e/op_geth.go

```diff
@@ -93,12 +106,16 @@ func NewOpGeth(t *testing.T, ctx context.Context, cfg *SystemConfig) (*OpGeth, e
 	l2Client, err := ethclient.Dial(node.HTTPEndpoint())
 	require.Nil(t, err)
 
+	l2RpcClient, err := rpc.Dial(node.HTTPEndpoint())
+	require.Nil(t, err)
```

Comment:
Should be able to just use:
```suggestion
	l2RpcClient := l2Client.Client()
```

Reply:
Actually, probably don't need to store the l2RpcClient separately since it can be retrieved from the ethclient.  That's a relatively new addition to the api but a very handy one.

Reply:
Yes! I removed `l2RpcClient` and retrieved it from `ethclient`.